src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/src/ansys/solutions/{{cookiecutter.__solution_name_slug}}/ui/utils/alerts.py

```diff
@@ -37,18 +37,28 @@ def update_alerts(problem_setup_step: ProblemSetupStep) -> list:
         )
 
     # optiSLang solve alert
-    if problem_setup_step.optislang_solve_status == "initial":
-        solve_message, solve_color = "optiSLang simulation not started.", "warning"
-    elif problem_setup_step.optislang_solve_status == "processing":
-        solve_message, solve_color = "optiSLang simulation in progress.", "primary"
-    elif problem_setup_step.optislang_solve_status == "stopped":
-        solve_message, solve_color = "optiSLang simulation stopped.", "danger"
-    elif problem_setup_step.optislang_solve_status == "aborted":
-        solve_message, solve_color = "optiSLang simulation aborted.", "danger"
-    elif problem_setup_step.optislang_solve_status == "finished":
-        solve_message, solve_color = "optiSLang simulation completed successfully.", "success"
+    if problem_setup_step.project_state == "NOT STARTED":
```

Comment:
Could this long `if` be changed to a dictionary instead?

Reply:
Done.